### PR TITLE
Review Guide

### LINKED ISSUE #132: Reviewing Pull Requests
As suggested by @sajayantony, it is good to [make a reviewing guide](https://contribute.cncf.io/maintainers/github/templates/recommended/reviewing/) for `oras` and `oras-go`.

Let's start with the `oras-go` repository, and do a copy in `oras` once the doc is revised.

### PR DESCRIPTION
**Pull request by shizhMSFT**
Resolves #132

### CODE REVIEW

File: REVIEWING.md
Comment (northtyphoon) [inline REVIEWING.md:42]: Is it for reviewer or author?

File: REVIEWING.md
Comment (northtyphoon) [inline REVIEWING.md:51]: Do we expect reviewer to run the check list?

File: REVIEWING.md
Comment (shizhMSFT) [reply REVIEWING.md:42]: The issue creation can be done anyone. For example, feature proposal or bug report.

File: REVIEWING.md
Comment (shizhMSFT) [reply REVIEWING.md:51]: Yes, we expect both the author and the reviewer to run the checklist.

File: REVIEWING.md
Comment (shizhMSFT) [reply REVIEWING.md:42]: > Is it for reviewer or author?

The answer is both.

File: REVIEWING.md
Comment (Wwwsylvia) [inline REVIEWING.md:38]: Can maintainers merge their own pull requests after it has been reviewed?  Or do they need to wait for other maintainers to merge it?

Comment (shizhMSFT) [reply REVIEWING.md:38]: Yes, they can merge if the PR is approved by other maintainers.

Comment (Wwwsylvia) [reply REVIEWING.md:38]: Should we add this into the process session?

**Comment by sajayantony** [comment]
@bridgetkromhout - would you be able to take a look at this?

File: REVIEWING.md
Comment (susanshi) [inline REVIEWING.md:35]: typo here from template "Only merge when _the_ change won't"

**Review by karenhchu — COMMENTED** [review]
One small edit. Otherwise it looks good!

File: REVIEWING.md
Comment (shizhMSFT) [reply REVIEWING.md]: fixed typo

File: REVIEWING.md
Comment (shizhMSFT) [reply REVIEWING.md]: fixed grammar

**Comment by shizhMSFT** [comment]
Pinging @jdolitsky @deitch @sajayantony for review.

**Review by sajayantony — APPROVED** [review]
Let' start with this for now.

**Review by sajayantony — APPROVED** [review]

**Pull request merged**
